tests/includes/server-timing/perflab-server-timing-tests.php

```diff
@@ -108,6 +108,7 @@ public function test_register_metric_requires_measure_callback() {
 
 		$this->server_timing->register_metric(
 			'metric-without-measure-callback',
+			// @phpstan-ignore-next-line
```

Comment:
Could you add note here why we ignore it?

Reply:
Good idea. Added in c588e588